### PR TITLE
broken-content script to include a new image tag

### PR DESCRIPTION
**Pull request by Vincent056**
We need to add the newly added image to the script, so that it will run on s390x

### CODE REVIEW

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ resources:
 
 images:
 - name: compliance-operator
-  newName: quay.io/compliance-operator/compliance-operator
+  newName: image-registry.openshift-image-registry.svc:5000/openshift/compliance-operator
```
Comment (mrogers950) [inline config/manager/kustomization.yaml:6]: @Vincent056 just a FYI, this substitution is local cause of the make targets, and shouldn't be committed. If you run `make clean` it'll restore the default.

**Review by rhmdnd — COMMENTED** [review]
Just a couple of comments inline. Also, can we add the corresponding bug (or jira?) to the commit message that includes additional context?

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Fixes
 
--
+- Fix broken-content script to include a new image tag
```
[the remark this answers is not in the document]
Comment (Vincent056) [reply CHANGELOG.md:16]: Sure thing, I think this script is used to test content on other arch

File: tests/e2e/e2e_test.go [hunk 3]
```diff
@@ -3676,7 +3676,7 @@ func TestE2E(t *testing.T) {
 			Name:       "TestCheckDefaultKubeletConfig",
 			IsParallel: true,
 			TestFn: func(t *testing.T, f *framework.Framework, ctx *framework.Context, mcTctx *mcTestCtx, namespace string) error {
-				var baselineImage = fmt.Sprintf("%s:%s", brokenContentImagePath, "kublet_default")
+				var baselineImage = fmt.Sprintf("%s:%s", brokenContentImagePath, "kubelet_default")
```
[the remark this answers is not in the document]
Comment (Vincent056) [reply tests/e2e/e2e_test.go:3679]: No, that was not the cause of the issue, let me add why in the changelog

**Review by rhmdnd — APPROVED** [review]
/lgtm

**Comment by Vincent056** [comment]
Since this is for testing and does not affect end user, add
/docs-approved
/px-approved

**Pull request merged**